go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/golang/mock v1.4.3
 	github.com/google/uuid v1.2.0
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
-	github.com/hashicorp/terraform-json v0.12.0 // indirect
+	github.com/hashicorp/terraform-json v0.13.0 // indirect
```

Comment:
had to make this upgrade
we had a problem where we got
```
Ran 9 of 9 Specs in 0.006 seconds
SUCCESS! -- 9 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestHttpClient (0.01s)
PASS
ok  	github.com/env0/terraform-provider-env0/client/http	0.018s
=== RUN   TestUnitConfigurationVariableData
=== RUN   TestUnitConfigurationVariableData/ScopeGlobal
    test_reporter.go:43: Step 1/2 error: unsupported state format version: expected ["0.1" "0.2"], got "1.0"
FAIL	github.com/env0/terraform-provider-env0/env0	4.159s
?   	github.com/env0/terraform-provider-env0/tests	[no test files]
?   	github.com/env0/terraform-provider-env0/utils	[no test files]
FAIL
```
and we fixed that as part of #132 
now i got it once again and the upgrade fixes that problem